build.sh

```diff
@@ -1,15 +1,15 @@
-#!/bin/bash
+#!/bin/bash -eu
 
-set -eu
+# Make sure we are in the directory this script is in.
+cd "$(dirname "$0")"
```

Comment:
Rather than `dirname` in a subshell, I prefer bash parameter expansion: `${0%/*}`.